README.md

```diff
@@ -10,6 +10,12 @@ runs on. You just have to press enter once! Be careful.
 
 #### Obtain a USB Image
 
+###### Configure your server
+
+    $ vi ocemr.yml
+
+Note: This is the ansible configuration file. See [ocemr_ansible](https://github.com/patfreeman/ocemr_ansible) for more information.
```

Comment:
I dig the ansible, why not just include it in this repo?

Reply:
Included now.